lib/wwtd.rb

```diff
@@ -136,9 +136,9 @@ def matrix_match?(cell, exclude)
       cell.values_at(*exclude.keys) == exclude.values
     end
 
-    def with_clean_env(&block)
+    def with_unbundled_env(&block)
       if defined?(Bundler)
-        Bundler.with_clean_env(&block)
+        Bundler.with_unbundled_env(&block)
```

Comment:
need to either do this or do a version check to not break old versions of bundler,
ideally also comment what version of bundler supports this if we ever mark any version of bundler deprecated
```
if Bundler.respond_to?(: with_unbundled_env)
  Bundler.with_unbundled_env(&block)
else
  Bundler.with_clean_env(&block)
end
```